cluster/charts/crossplane/templates/environment-personas-clusterroles.yaml

```diff
@@ -0,0 +1,180 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-env-admin
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-environment-admin: "true"
+rules: [] # Rules are automatically filled in by the controller manager.
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:env:default:admin
+  labels:
+    rbac.crossplane.io/aggregate-to-environment-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - stacks.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - delete
+  - deletecollection
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-env-edit
+  labels:
+    rbac.crossplane.io/aggregate-to-environment-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-environment-edit: "true"
+rules: [] # Rules are automatically filled in by the controller manager.
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:env:default:edit
+  labels:
+    rbac.crossplane.io/aggregate-to-environment-edit: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - stacks.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - create
```

Comment:
should this set of verbs match what's listed in https://github.com/displague/crossplane/blob/gh1049-stack-controllers-use-clusterrole/pkg/controller/stacks/stack/stack.go#L61?

Reply:
The shorter list of verbs are provided because these `default` roles are only intended for internal user.  The `-edit` and `-admin` clusterroles pick up verbs from both the default `view` and `edit` clusterroles.

The verbs are cumulative.